playground/index-node.js

```diff
@@ -0,0 +1,8 @@
+console.log('hey node')
+// require('isomorphic-fetch')
+
+const Client = require('../')
+const { RichtextInstance } = require('../')
+const cli = new Client({})
+console.log(cli.get)
+console.log('Imported')
```

Comment:
We need this logs here too ? and the comment ?

Reply:
same